gather/views.py

```diff
@@ -9,16 +9,13 @@
 from django.contrib.auth.forms import AuthenticationForm, UserCreationForm
 from django.http import HttpResponse, HttpResponseRedirect
 from django.shortcuts import render, render_to_response
-from django.template import RequestContext
 from django.contrib import messages
 from django.conf import settings
 from django.db.models import Q
 from gather.models import Event, EventAdminGroup, EventSeries
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
-from django.contrib import messages
 from django.views.decorators.csrf import csrf_exempt
 from django.template.loader import get_template
-from django.template import Context
```

Comment:
Heh. Running linters/static analysis over the whole codebase would probably be a productive way to clean it up...